src/Superstructure/ESMFMod/interface/ESMF_Init_C.F90

```diff
@@ -59,14 +59,26 @@ subroutine f_esmf_frameworkinitialize(lang, configFileName, &
 
    end subroutine f_esmf_frameworkinitialize
 
-   subroutine f_esmf_frameworkfinalize(rc)
+   subroutine f_esmf_frameworkfinalize(rc, keepMpi)
        use ESMF_CompMod
        use ESMF_InitMod
+       use ESMF_UtilTypesMod
+       use, intrinsic :: iso_c_binding
```

Comment:
Just realized a mistake I made -- this implementation is F90, so I don't believe that I can use `iso_c_binding` here (if I remember correctly, it's only F03 and up). May need to pass in an integer instead.

Reply:
It's fine to use iso_c_binding. Despite the file extension being F90, we support this and other F03 features. (I see we use iso_c_binding throughout ESMF, though I'll let @oehmke comment on whether there's a different typical/preferred approach.)

Reply:
We're slowly moving to iso_c_binding as things are changed or new code is added, so I agree that using it is a good approach.